src/summary/ScatterPlotVisualizer.jsx

```diff
@@ -39,246 +39,249 @@ class ScatterPlotVisualizer extends Component {
+                    labels: {
+                        style: {
+                            fontSize: '13px'
+                        },
+                        y: 30,
+                        useHTML: true,
+                        formatter: function () {
+                            const index = this.axis.categories.indexOf(this.value);
+                            let sum = 0;
+                            myData.forEach(function (series) {
+                                series.forEach(function (data) {
+                                    if (data[0] === index)
+                                        sum += 1;
+                                })
+                            });
+                            return '<span><font color = "#3AA7F0">' + sum + '</font >' +
+                                '<br>' + this.value;
+                        }
                     }
-                }
-            }],
-            yAxis: {
-                min: 0,
-                max: 80.1,
-                title: {
-                    text: 'Surivival time (total # of months)',
-                    style: {
-                        fontSize: '15px'
+                }, { //second x axis
+                    opposite: true,
+                    linkedTo: 0,
+                    title: {
+                        text: '<span>Patients who <br> survived at least <br> 60 months</span>',
+                        useHTML: true,
+                        floating: true,
+                        align: 'left',
+                        margin: -20,
+                        y: 20,
+                        style: {
+                            color: '#3AA7F0',
+                            fontSize: '13px'
+                        }
+                    },
+                    categories: ['Chemo Therapy', 'Chemo therapy & Radiation', 'Hormonal Therapy', 'Radiation', 'Surgery', 'Surgery & Radiation', 'None (Actively Monitoring)'],
+                    labels: {
+                        style: {
+                            fontSize: '13px'
+                        },
+                        useHTML: true,
+                        formatter: function () {
+                            const index = this.axis.categories.indexOf(this.value);
+                            let categorySum = 0;
+                            let livingOver60 = 0;
+                            myData.forEach(function (series) {
+                                series.forEach(function (data) {
+                                    if (data[0] === index) {
+                                        categorySum += 1;
+                                        if (data[0] === index && data[1] >= 60)
+                                            livingOver60 += 1;
+                                    }
+                                })
+                            });
+                            return '<span><font color= "#3AA7F0">' + livingOver60 + '<br>'
```

Comment:
[MG3] This can be changed to use backtick notation. I.e. return `<span><font ...>{livingOver60}<br>${Highcharts.format...` etc.